processor/PlanExecutor_test.go

```diff
@@ -50,6 +51,12 @@ var _ = Describe("Plan Executor", func() {
 		configuration.FilePath = file.Name()
 		bar = NullProgressBar{}
 		aggregator = statistics.NewAggregator(metrics.DefaultRegistry)
+		channel = make(chan core.ExecutionResult)
+
+		go func() {
+			for range channel {
+			}
```

Comment:
Thing is though if I get the value of numberOfMessages wrong, the write operation to the channel will block.  With this way it won't.